runtime/vm/JFRConstantPoolTypes.hpp

```diff
@@ -1209,6 +1226,26 @@ class VM_JFRConstantPoolTypes {
 		}
 	}
 
+	/**
+	 * Initialize GCHeapConfigurationEntry
+	 *
+	 * @param vm[in] the J9JavaVM
+	 */
+	static void initializeGCHeapConfigurationEvent(J9JavaVM *vm, BuildResult *result)
```

Comment:
I dont think we need this function, we can write the values directly in `writeGCHeapConfigurationEvent`